daemon/api_users.go

```diff
@@ -472,6 +485,10 @@ type postUserCreateData struct {
 	Sudoer       bool   `json:"sudoer"`
 	Known        bool   `json:"known"`
 	ForceManaged bool   `json:"force-managed"`
+
+	// not from JSON: this indicates whether to preserve backwards
+	// compatibility, which results in more clunky return values
+	compat bool
```

Comment:
singleUserResultCompat perhaps, also I would still use

<field name> indicates ...  ; not from JSON

Reply:
done